website/www/site/layouts/get-started/baseof.html

```diff
@@ -37,5 +37,6 @@
     </div>
   </div>
 {{ partial "footer.html" . }}
+{{ partial "check_for_playground.html" . }}
```

Comment:
This only adds this check to `get-started`, but not the home page. Maybe better add this to `footer.html`? That will cover all pages on the website.